app/templates/login.hbs

```diff
@@ -94,28 +94,30 @@
     onClose=closeModal
     translucentOverlay=true
   }}
-    <h2>Forgot password</h2>
+    <h2 class="modal-header">Forgot password</h2>
     {{#let forgotPassword as |task|}}
       <form onsubmit={{prevent-default (perform task)}}>
         <legend>
-          First we need the e-mail address on your account
+          Enter your email address below and we'll email the instruction for setting a new one.
         </legend>
         <div>
-          <label>email address:</label>
+          {{!-- <label>Email address:</label> --}}
           <input
             type="email"
             required
-            oninput={{action (mut modalContext.email) value="target.value"}}>
+            oninput={{action (mut modalContext.email)
+            value="target.value"}}
```

Comment:
This `value="target.value"` is part of the `{{action ...}}`, so it should be back on the previous line so it's easier to read.

Reply:
Changed back.